tests/python/unittest/test_runtime_micro.py

```diff
@@ -25,8 +25,25 @@
 from tvm.micro import create_micro_mod
 from tvm.relay.testing import resnet
 
-# Use the host emulated micro device.
-DEV_CONFIG = micro.device.host.default_config()
+# # Use the host emulated micro device.
+DEV_CONFIG_A = micro.device.host.generate_config()
+DEV_CONFIG_B = micro.device.host.generate_config()
+TARGET = 'c -device=micro_dev'
+
+# # TODO why do spike examples have memory that starts at 0x10000000, but you
```

Comment:
Do we need to keep these lines commented out?

Reply:
i'm not sure what they were there for. it seemed like a thing we would hit again, but we can always pull them from the demo tag. deleted.